packages/contracts-bedrock/scripts/ChainAssertions.sol

```diff
@@ -38,26 +38,38 @@ library ChainAssertions {
         ResourceMetering.ResourceConfig memory dflt = Constants.DEFAULT_RESOURCE_CONFIG();
         require(keccak256(abi.encode(rcfg)) == keccak256(abi.encode(dflt)));
 
-        checkSystemConfig({ _contracts: _prox, _cfg: _cfg, _isProxy: true });
+        checkSystemConfig({ _contracts: _prox, _cfg: _cfg, _vm: _vm, _isProxy: true });
         checkL1CrossDomainMessenger({ _contracts: _prox, _vm: _vm, _isProxy: true });
-        checkL1StandardBridge({ _contracts: _prox, _isProxy: true });
+        checkL1StandardBridge({ _contracts: _prox, _vm: _vm, _isProxy: true });
         checkL2OutputOracle({
             _contracts: _prox,
             _cfg: _cfg,
             _l2OutputOracleStartingTimestamp: _l2OutputOracleStartingTimestamp,
+            _vm: _vm,
             _isProxy: true
         });
-        checkOptimismMintableERC20Factory({ _contracts: _prox, _isProxy: true });
-        checkL1ERC721Bridge({ _contracts: _prox, _isProxy: true });
-        checkOptimismPortal({ _contracts: _prox, _cfg: _cfg, _isProxy: true });
-        checkProtocolVersions({ _contracts: _prox, _cfg: _cfg, _isProxy: true });
+        checkOptimismMintableERC20Factory({ _contracts: _prox, _vm: _vm, _isProxy: true });
+        checkL1ERC721Bridge({ _contracts: _prox, _vm: _vm, _isProxy: true });
+        checkOptimismPortal({ _contracts: _prox, _cfg: _cfg, _vm: _vm, _isProxy: true });
+        checkProtocolVersions({ _contracts: _prox, _cfg: _cfg, _vm: _vm, _isProxy: true });
     }
 
     /// @notice Asserts that the SystemConfig is setup correctly
-    function checkSystemConfig(Types.ContractSet memory _contracts, DeployConfig _cfg, bool _isProxy) internal view {
+    function checkSystemConfig(
+        Types.ContractSet memory _contracts,
+        DeployConfig _cfg,
+        Vm _vm,
+        bool _isProxy
+    )
+        internal
+        view
+    {
         console.log("Running chain assertions on the SystemConfig");
         SystemConfig config = SystemConfig(_contracts.SystemConfig);
 
+        bytes32 initialized = _vm.load(address(config), bytes32(0));
```

Comment:
There is a function someplace for getting the initialized slot, we should extract that function into a place where it can be used in here and where it is already used

Reply:
+1, I was going to say pull the slot into a constant, but your way is even better. I found the method: `function loadInitializedSlot` in `Deployer.sol`